packages/skeleton-svelte/src/lib/components/Combobox/Combobox.svelte

```diff
@@ -93,7 +93,7 @@
 			<!-- Input -->
 			<input {...api.getInputProps()} class={inputGroupInput} />
 			<!-- Arrow -->
-			<button {...api.getTriggerProps()} class={inputGroupButton}>
+			<button {...api.getTriggerProps()} class={inputGroupButton} type="button">
```

Comment:
Long, actually after seeing this I was reminded Zag typically appends all the required attributes. So I looked and sure enough this type is already set, so this is redudant:

![image](https://github.com/user-attachments/assets/af7a2908-fa28-4865-9a46-3223dcb56bd3)

There much be another cause for why this is submitting a form. Feel free to research on your own, otherwise I'll need to circle back to this.

Reply:
I'll look into it.

Reply:
I actually don't have that. On docs site:

![image](https://github.com/user-attachments/assets/3e3084c4-8a40-4336-b342-d12ea68b41a5)

Could this be browser behavior? I'm using Chrome 131.0.6778.86 (Official Build) (64-bit) (cohort: Stable)

Reply:
Ok I realized what's going on - I was inspecting the target input's button. You're reviewing the drop-down menu. What you're showing is populated in the "Content" section of the markup:

https://github.com/skeletonlabs/skeleton/blob/d0ee088343f8b59ffb6c29487c5a8c73d367048f/packages/skeleton-svelte/src/lib/components/Combobox/Combobox.svelte#L129

So i can confirm the type attribute is missing and will need to be appended by us for now. Really though, this issue should reported upstream and patch on Zag's end. Because this goes outside their standard behavior here. They apply the type for all other button elements.

Reply:
Could you do the reporting upstream to Zag? My understanding to their product is quite limited so I will most likely not know what I'm talking about had I filed an issue. I'll put a comment to the mark that links to your comment.

Reply:
Sure